src/draw/sw/lv_draw_sw_rect.c

```diff
@@ -19,9 +19,24 @@
 #define SHADOW_ENHANCE          1
 #define SPLIT_LIMIT             50
 
+#define LV_DITHER_GRADIENT 1
```

Comment:
It should be in `lv_conf.h`.

Reply:
Yes. On the last commit only, since it'll break CI if I change it, right?